.github/workflows/trigger-chromium-build.yml

```diff
@@ -0,0 +1,129 @@
+name: Trigger Chromium Build
+
+on:
+  issues:
+    types: [labeled, reopened]
+
+permissions:
+  contents: read
+  issues: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  matches_label:
+    if: | 
+      github.event.label.name == 'trigger-chromium-build'
+      && github.event.issue.author_association == 'MEMBER'
+    runs-on: ubuntu-latest
+    outputs:
+      version_bump_config: ${{ steps.extract_version_bump_config.outputs.result }}
+    steps:
+      - name: Install Dependencies
+        run: |
+          apt-get update && apt-get install jq wget -y
+      - name: Install mdsh
+        run: |
+          wget https://github.com/bashup/mdsh/raw/master/bin/mdsh && \
+          chmod +x mdsh && \
+          export PATH=$PATH:mdsh
+      - name: Extract version bump configuration
+        id: extract_version_bump_config
+        run: |
+          # mdsh-* functions are informed from https://github.com/bashup/mdsh/blob/master/mdsh.md 
+
```

Comment:
We only leverage the parse functionality provided by mdsh, so no arbitrary code will be executed from the issue created